relnotes/stats.deprecation.md

```diff
@@ -0,0 +1,8 @@
+### `IRequestStats.RequestStats.mean_handled_time_micros` renamed
+
+`swarm.neo.client.mixins.ClientCore`, `swarm.neo.client.requests.Stats`
+
+The method `IRequestStats.RequestStats.mean_handled_time_micros` has been
+deprecated. Usage should be replaced with
+`IRequestStats.RequestStats.total_time_micros`.
```

Comment:
The suggested replacement is wrong, it should be `IRequestStats.RequestStats.histogram.mean_time_micros`, no?

Reply:
No, there's also a wrapper method for that. Then we'll be able to cleanly replace `RequestStats` with `TimeHistogram`.

Reply:
Oh it should be `IRequestStats.RequestStats.mean_time_micros`, though, of course (without the `histogram` in the middle).

Reply:
Updated again.